### ISSUE TITLE
typo under "Websites using SkipTo.js" heading

### ISSUE THREAD

**Issue by jdav-illinois** [skipto-landmarks-headings/page-script-5#23]
in the last sentence of the section, "The “popup” option is the more traditional approach to fulfilling the “[bypass bocks](https://www.w3.org/TR/WCAG/#bypass-blocks)” requirement of the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG/), but this option makes the feature less visible to people who might benefit." 

"bypass blocks" is missing the L in "blocks".

**Comment by jongund** [comment]
Thank you
